neo/Wallets/Wallet.py

```diff
@@ -894,16 +894,21 @@ def GetDefaultContract(self):
         Get the default contract.
 
         Returns:
-            contract (Contract): if Successful, a contract of type neo.SmartContract.Contract, otherwise None
+            contract (Contract): if Successful, a contract of type neo.SmartContract.Contract, otherwise an Exception.
+
+        Raises:
+            Exception: if no default contract is found.
 
         Note:
             Prints a warning to the console if the default contract could not be found.
         """
         try:
-            return self.GetContracts()[0]
-        except Exception as e:
-            logger.error("Could not find default contract")
-        return None
+            default = self.GetContracts()[0]
```

Comment:
here you could get rid of the `default` variable and just straight `return self.GetContracts...`

Reply:
yeah, I guess it's a habit to try and not have multiple returns in a function.

Reply:
It's actually a good pattern to return early, when you're done early! 

Makes it easier to keep track of what's happening and what has already ended, and often times you need less indentation.